spec/lita/clients/harvest_spec.rb

```diff
@@ -1,24 +1,39 @@
-require "spec_helper"
+require 'spec_helper'
 
 describe Lita::Clients::Harvest do
-  let(:client) do 
+  let(:client) do
     described_class.new ENV.fetch('HARVEST_ACCOUNT_ID', 'XXX'), ENV.fetch('HARVEST_TOKEN', 'XXX')
   end
 
-  before do 
-    allow_any_instance_of(described_class).to receive(:to_param).and_return("?to=2017-10-20T23:59:59Z")
+  before do
+    allow_any_instance_of(described_class).to \
+      receive(:to_param).and_return('?to=2018-12-24T23:59:59Z')
   end
 
-  describe "#users_current_entries", vcr: { cassette_name: 'harvest-entries' } do 
-    let!(:entries) { client.users_current_entries }
-    it { expect(entries[3].owner).to eq("Mario López") }
-    it { expect(entries[3].description).to eq("Programming: lita_time_tracker") }
-    it { expect(entries[3].user_email).to eq("mario@platan.us") }
-  end
+  describe '#users_current_entries' do
+    context 'with no inactive users', vcr: { cassette_name: 'harvest-entries-active-users' } do
+      let!(:entries) { client.users_current_entries }
```

Comment:
la diferencia entre `let!` y `let` es que el segundo se usa como un proc. Es decir, se ejecuta cuando lo llamás. En este caso vos estás llamando a `entries` en cada `it` por lo que el `let!` no te hace falta. Usas `let!`cuando querés armar contexto alrededor de tu test. Es el equivalente en poner algo en un `before`

Reply:
Perfecto! Me quedo mucho más claro
La tercera es la vencida